HopsanGUI/HcomHandler.cpp

```diff
@@ -7261,7 +7261,7 @@ void HcomHandler::evaluateExpression(QString expr, VariableType desiredType)
             isHcomFunctionCall("removeDivisors", expr) || isHcomFunctionCall("left", expr) ||
             isHcomFunctionCall("right", expr) || isHcomFunctionCall("trapezoid", expr) ||
             isHcomFunctionCall("euler", expr) || isHcomFunctionCall("bdf1", expr) ||
-            isHcomFunctionCall("bdf2", expr)) {
+            isHcomFunctionCall("bdf2", expr) || isHcomFunctionCall("latex", expr)) {
```

Comment:
Are any of these symhop/hcom functions part of the hcom documentation?

Reply:
No, you are right. They should be registered as internal functions. I will fix that.